src/app.js

```
@@ -165,26 +172,22 @@ const telemetryConfig = {
 
 telemetry.init(telemetryConfig)
 
-// function to update the config
-function updateConfig (configString) {
-  configUtil.setConfig('CHANNEL_FILTER_QUERY_STRING', configString)
+// function to generate the search filter and return JSON Object
+function getMetaFilterConfig () {
+// Check if the Filter Config service data is defined, if yes, create Object with it
+  if (filterConfigService !== undefined) {
+    return getFilterJSONfromConfigService()
+  } else {
+  // Call getFilterJSONFromEnv to generate a JSON Object
+    return getFilterJSONFromEnv()
+  }
 }
 
-// function to generate the search string
-function getFilterConfig () {
-  LOG.info(utilsService.getLoggerData({}, 'INFO',
-    filename, 'getFilterConfig', 'environment info', process.env))
-  var allowedChannels = whiteListedChannelList ? whiteListedChannelList.split(',') : []
-  var blackListedChannels = blackListedChannelList ? blackListedChannelList.split(',') : []
-  var configString = {}
-  if ((allowedChannels && allowedChannels.length > 0) && (blackListedChannels && blackListedChannels.length > 0)) {
-    configString = _.difference(allowedChannels, blackListedChannels)
-  } else if (allowedChannels && allowedChannels.length > 0) {
-    configString = allowedChannels
-  } else if (blackListedChannels && blackListedChannels.length > 0) {
-    configString = { 'ne': blackListedChannels }
-  }
-  LOG.info(utilsService.getLoggerData({}, 'INFO',
-    filename, 'getFilterConfig', 'config string', configString))
-  return configString
+function getFilterJSONFromEnv () {
+  // Generate JSON and return
+  return null
+}
+
+function getFilterJSONfromConfigService () {
```

this should raise an exception since it is not implemented.